content/docs/esc/integrations/infrastructure/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Infrastructure
```

Comment:
This heading feels a bit vague, and doesn't quite line up with the content underneath it (i.e., integrationg with Pulumi IaC, the Terraform CLI, and various aspects of CloudFlare and CloudFlare-related tooling).

Reply:
Plan will be to leave this category title for now, assess after next week